modules/Core/includes/endpoints/GetNotificationsEndpoint.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @see Alert
+ * 
+ * @param int $id The NamelessMC user to get notifications for
+ *
+ * @return string JSON Array
+ */
+class GetNotificationsEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'getNotifications';
+        $this->_module = 'Core';
+        $this->_description = 'Get notifications for a user';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.